static/app/views/issueDetails/streamline/eventTitle.tsx

```diff
@@ -118,51 +117,41 @@ export const EventTitle = forwardRef<HTMLDivElement, EventNavigationProps>(
       <div {...props} ref={ref}>
         <EventInfoJumpToWrapper>
           <EventInfo>
-            <EventIdInfo>
-              <DropdownMenu
-                trigger={(triggerProps, isOpen) => (
-                  // Tooltip split from button to prevent re-opening w/ focus event on close
-                  <Tooltip
-                    title={event.id}
-                    delay={500}
-                    overlayStyle={{maxWidth: 'max-content'}}
-                    disabled={isOpen}
-                  >
```

Comment:
removing the tooltip for now as that was causing issues with the dropdown button when disabled

need to followup with a fix on dropdown buttons with tooltips. they shouldn't trigger the tooltip on action